client/my-sites/email/email-management/titan-management-nav/index.jsx

```diff
@@ -78,39 +87,48 @@ class TitanManagementNav extends React.Component {
 		);
 	};
 
-	renderAddMailboxesLink = () => {
+	render() {
 		const { currentRoute, domain, selectedSiteSlug, translate } = this.props;
-		// If we don't have a subscription, we can't add mailboxes.
-		if ( ! domain?.titanMailSubscription?.subscriptionId ) {
-			return null;
-		}
 
-		return (
-			<VerticalNavItem
-				path={ emailManagementNewTitanAccount( selectedSiteSlug, domain.name, currentRoute ) }
+		const header = (
+			<>
+				<Gridicon icon="my-sites" size={ 36 } />
+
+				<span className="titan-management-nav__foldable-card-main-text">
+					<strong>{ getTitanProductName() }</strong>
+					<em>{ domain.name }</em>
+				</span>
+			</>
+		);
+
+		const summary = domain?.titanMailSubscription?.subscriptionId && (
+			<Button
+				primary
+				compact
+				href={ emailManagementNewTitanAccount( selectedSiteSlug, domain.name, currentRoute ) }
 			>
-				{ translate( 'Add more mailboxes' ) }
-			</VerticalNavItem>
+				{ translate( 'Add New Mailboxes' ) }
+			</Button>
 		);
-	};
 
-	render() {
-		const { domain, translate } = this.props;
-		const headerLabel = translate( '%(productName)s: %(domainName)s', {
-			args: {
-				domainName: domain.name,
-				productName: getTitanProductName(),
-			},
-			comment:
-				'%(domainName)s is a domain name, e.g. example.com; %(productName)s is the product name, either "Email" or "Titan Mail"',
-		} );
 		return (
 			<div className="titan-management-nav">
-				<SectionHeader label={ headerLabel } />
+				<FoldableCard
+					className="titan-management-nav__foldable-card"
+					header={ header }
+					summary={ summary }
+					expandedSummary={ summary }
+				>
+					<ul className="titan-management-nav__foldable-card-services">
+						<li><a href="https://wp.titan.email/mail/" target="_blank" rel="noreferrer noopener"><img src={ titanMailIcon } alt={ translate( 'Titan Mail icon' ) } /><strong>{ translate( 'Mail' ) }</strong></a></li>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **40** times:
`translate( 'mail' )` *ES Score: 9*